src/patternfly/components/Dropdown/styles.scss

```diff
@@ -0,0 +1,158 @@
+
+  &__toggle {
+    padding: $pf-c-dropdown__toggle--PaddingTop $pf-c-dropdown__toggle--PaddingRight $pf-c-dropdown__toggle--PaddingBottom $pf-c-dropdown__toggle--PaddingLeft;
+    font-size: $pf-c-dropdown__toggle--FontSize;
+    color: $pf-c-dropdown__toggle--Color;
+    background-color: $pf-c-dropdown__toggle--BackgroundColor;
+    border: $pf-c-dropdown__toggle--BorderWidth solid $pf-c-dropdown__toggle--BorderColor;
+
+    &:hover {
+      padding-top: calc(#{$pf-c-dropdown__toggle--PaddingTop} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-right: calc(#{$pf-c-dropdown__toggle--PaddingRight} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-bottom: calc(#{$pf-c-dropdown__toggle--PaddingBottom} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-left: calc(#{$pf-c-dropdown__toggle--PaddingLeft} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      border-width: $pf-c-dropdown__toggle--hover--BorderWidth;
+    }
+
+    .pf-is-expanded & {
+      padding-top: calc(#{$pf-c-dropdown__toggle--PaddingTop} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-right: calc(#{$pf-c-dropdown__toggle--PaddingRight} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-bottom: calc(#{$pf-c-dropdown__toggle--PaddingBottom} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-left: calc(#{$pf-c-dropdown__toggle--PaddingLeft} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      border-color: $pf-c-dropdown__toggle--is-expanded--BorderColor;
+      border-width: $pf-c-dropdown__toggle--is-expanded--BorderWidth;
+    }
+
+    .pf-is-ghost & {
+      background-color: transparent;
+      border-color: transparent;
+    }
+
+    .pf-sans-arrow & {
+      padding-right: $pf-c-dropdown__toggle--PaddingLeft;
+    }
+
+    &::after {
+      box-sizing: content-box;
+      display: inline-block;
+      width: calc(#{$pf-c-dropdown__toggle--FontSize} * .3);
+      height: calc(#{$pf-c-dropdown__toggle--FontSize} * .3);
+      margin-left: $pf-c-dropdown__arrow--MarginLeft;
+      vertical-align: calc(#{$pf-c-dropdown__toggle--FontSize} * .25);
+      content: "";
+      border-style: solid;
+      border-width: $pf-c-dropdown__arrow--BorderWidth $pf-c-dropdown__arrow--BorderWidth 0 0;
+      transform: rotate(135deg);
+
+      .pf-sans-arrow & {
+        content: none;
+      }
+
+      .pf-is-dropup.pf-is-expanded & {
+        vertical-align: calc(#{$pf-c-dropdown__toggle--FontSize} * .1);
+        transform: rotate(-45deg);
+      }
+    }
+  }
+
+  &__menu {
+    position: absolute;
+    z-index: 1000;
```

Comment:
Does PF not prescribe a set of elevation variables? If not, I think it's worth looking into.

Reply:
@andresgalante did you have something in mind that you'd like me to add?

Should I just do: `$pf-global--ZIndex:    1000;` for now, and we update them later?

Reply:
@jgiardino if you are doing that please create an issue for us to add zindex global variables later. If not just add them :)